build/constants.yaml

```diff
@@ -1,19 +1,19 @@
 - name: ext-var-names
```

Comment:
Only a format change. The C# `YamlDotNet` library removes comments, removes double quotes, has specific spacing requirements, and converts list with `[]` to `-`. Example: https://github.com/microsoft/Oryx/pull/1560/files#diff-47c28d7a6c8135707f46b624b5913e35beea6dfbe7a8be2db7efefde606eba59R47

Reply:
Correct. There is no simple approach on saving the comments. The good thing is that this should only be modified by Automation

Reply:
That isn't the case though, right? Whenever we want to create a new constant that spans our scripts and go files and csharp projects, this is the place we want to create that constant so its consistent, which would require manually editing this file? Also, we keep a lot of settings and environment variable information in here as well, which could require manual editing in the future unless I"m mistaken

Reply:
Yeah perhaps there can still be minor editing after all automation is still in place. If missing comments are critical, I curretnly see a couple options which would be to have a separate doc with comments for this, or we can have 2 constants.yaml one that can be manually updated with comments and one that is automatically updated without comments.